queryable-encryption/go/aws/reader/make-data-key.go

```diff
@@ -169,7 +169,7 @@ func MakeKey() error {
 		return fmt.Errorf("Error creating collection: %v", err)
 	}
 	// end-create-enc-collection
-	print("Created Encrypted Collection")
+	print("Created Encrypted Collection\n")
```

Comment:
I'm not sure printx is included in the [fmt](https://pkg.go.dev/fmt) package. Is there something else you are referring to? I'll update this to `fmt.Println` in the meantime.

It might make sense to let the DBX go team make any additional changes to this script as part of there upcoming technical review.